src/views/TreeView.ts

```diff
@@ -135,7 +144,6 @@ class Tag extends vscode.TreeItem {
 class Measurement extends vscode.TreeItem {
     constructor(
         private connection : IConnection,
-        private context : vscode.ExtensionContext,
```

Comment:
Was this context just... never used in the Tag/Measurement/Bucket/etc classes? Yikes.

Reply:
No, it was, until the `Store` model became the way to access data. After this patch, the `Store` holds a reference to the `ExtensionContext`, and so those other classes don't need it anymore.

Having to pass the `context` to every little object was kinda the driving thought behind this work, and I'm going to continue finding ways to encapsulate functionality like this because it really makes for a cleaner interface.